dockerfiles/alpine_3.5_1.x

```diff
@@ -17,17 +17,17 @@ ENV TARANTOOL_VERSION=${TNT_VER} \
     LUAROCKS_URL=https://github.com/tarantool/luarocks/archive/6e6fe62d9409fe2103c0fd091cccb3da0451faf5.tar.gz \
```

Comment:
Below you upgrade some rocks in major releases (avro-schema 2 -> 3) it could break backward compatibility. But I think that it's ok.

So I suggest you to drop "shard" support and add "vshard" instead. As for now we ship unsupported modules to some users. My main concern that docker images are quite popular for newcomer and it's quite strange to ship deprecated modules for them.

Reply:
Sure, I've made this change in the additional commit.